src/taco-cli/cli/run.ts

```diff
@@ -177,38 +177,30 @@ class Run extends commands.TacoCommandBase {
 
     /* tslint:disable:member-ordering */
```

Comment:
How that the other methods are not longer static, can this rule be deactivated and this member be put at the top of the file?